<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWidget</class>
 <widget class="QWidget" name="ProfileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hLayoutProfile">
   <item>
    <layout class="QVBoxLayout" name="vLayoutGroup1"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutGroup2"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutGroup3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutGroup4"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAbility">
     <property name="title">
      <string>특성</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="hLayoutAbility1"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayoutAbility2"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
